### PR TITLE
Nbenton/orc 460/initial commit

### PR DESCRIPTION
**Pull request by NonpareilNic**

### CODE REVIEW

**Comment by NonpareilNic** [comment]
Realizing late in the PR that I made the branch ORC-460 which is for Domain, this should be https://linear.app/massdriver/issue/ORC-461/create-bundle-for-aws-sagemaker-inference-endpoint

File: src/kms.tf [hunk 1]
```diff
@@ -0,0 +1,47 @@
+// Policy from https://docs.aws.amazon.com/efs/latest/ug/encryption-at-rest.html
+data "aws_iam_policy_document" "efs" {
```
[the remark this answers is not in the document]
Comment (NonpareilNic) [reply src/kms.tf:2]: Constraints added to all but ECR.

File: src/security_group.tf [hunk 2]
```diff
@@ -0,0 +1,21 @@
+resource "aws_security_group" "sagemaker_user" {
```
[the remark this answers is not in the document]
Comment (NonpareilNic) [reply src/security_group.tf:1]: SageMaker attaches SG to ENI behind the scenes. This SG is set as the User's default SG.

File: massdriver.yaml [hunk 3]
```diff
@@ -0,0 +1,51 @@
...
+  properties:
+    efs:
+      type: object
+      title: EFS Volume
+      required:
+        - retention_policy
+      properties:
+        retention_policy:
+          title: EFS Retention Policy
+          description: "The EFS Retention Policy. (e.g. Delete or Retain)"
```
Comment (chrisghill) [inline massdriver.yaml]:
```suggestion
          description: "The EFS Retention Policy. This determines what happens to the EFS volume when this bundle is decommissioned. (e.g. Delete or Retain)"
```

**Review by chrisghill — APPROVED** [review]

**Pull request merged**